dht/dht.go

```diff
@@ -980,11 +980,13 @@ func bootstrapAddrs(nodeAddrs []string) (addrs []*net.UDPAddr, err error) {
 		}
 	}
 	for _, addrStr := range bootstrapNodes {
-		udpAddr, err := net.ResolveUDPAddr("udp4", addrStr)
-		if err != nil {
-			continue
+		if addrStr != "" {
```

Comment:
I agree with this. It is an ugly hack :-) The reason why I have added the check is, when you are starting your own bittorrent DHT and want to avoid the external ones (ie. utorrent.com and bittorrent.com), currently the ```bootstrapNodes``` must have at least one item.

I need to think about a way to start the "private/internal" bittorrent DHT in some cleaner way than this terrible hack :+1: 
Do you have any suggestions here ?